identity-service/src/routes/solana.js

```diff
@@ -130,6 +136,25 @@ solanaRouter.post('/relay/raw', handleResponse(async (req, res, next) => {
   }
 }))
 
+solanaRouter.get('/random_fee_payer', handleResponse(async () => {
+  const solanaFeePayerWallets = config.get('solanaFeePayerWallets')
+  if (!solanaFeePayerWallets || !solanaFeePayerWallets.length) {
+    // if there is no list of fee payers, then
+    // check for single fee payer and return its public key if it exists
+    const solanaFeePayerWallet = config.get('solanaFeePayerWallet')
+    if (solanaFeePayerWallet) {
+      const feePayer = solanaWeb3.Keypair.fromSecretKey(Uint8Array.from(solanaFeePayerWallet))
+      return successResponse({ feePayer: feePayer.publicKey })
+    }
+
+    return errorResponseServerError('There is no list of fee payers to choose from.')
+  }
+
+  const randomFeePayerIndex = Math.floor(Math.random() * solanaFeePayerWallets.length)
```

Comment:
can we consolidate this logic with getFeePayer in solana-client.js? one place would be great

Reply:
https://github.com/AudiusProject/audius-protocol/blob/master/identity-service/src/solana-client.js#L100